### DISCUSSION TITLE
Bunyan Logging

### DISCUSSION THREAD

**Discussion by Zambonilli** [jsforce/jsforce#1436] [Ideas]
Is there any way other than the client overwriting the connection._logger or overwriting console to have jsforce use bunyan as opposed to the console?  It'd be nice to have this generic so logging can use the package of the client's choice.

This is something I could do a PR for if you want.

**Comment by stomita** [comment]
The logger used in JSforce is not so much extensible right now. You can send PR as what you want to be like.
